### PR TITLE
Paperless ngx

### PR DESCRIPTION
**Pull request by andreasbrett**
Add [Paperless-ngx](https://github.com/paperless-ngx/paperless-ngx) collection incl. parser and scenarios to the hub

### CODE REVIEW

File: .tests/paperless-ngx-logs/parser.assert
Comment (LaurenceJJones) [inline .tests/paperless-ngx-logs/parser.assert:1]: Could you include all of the parser assert output not just for s01-parse stage

**Review by LaurenceJJones — CHANGES_REQUESTED** [review]
Other than single comment once that is done, can be merged

Also thank you for spending the time making this! 🎆

File: .tests/paperless-ngx-logs/parser.assert
Comment (andreasbrett) [reply .tests/paperless-ngx-logs/parser.assert:1]: fair point. latest commit includes all the parser assert output.

**Review by LaurenceJJones — APPROVED** [review]
LGTM flakey tests (outside scope of this PR)

**Pull request merged**